pkg/cmd/addon/disable/exec.go

```diff
@@ -0,0 +1,114 @@
+// Copyright Contributors to the Open Cluster Management project
+package disable
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+
+	"github.com/spf13/cobra"
+	apiextensionsclient "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog/v2"
+	addonclient "open-cluster-management.io/api/client/addon/clientset/versioned"
+	clusterclientset "open-cluster-management.io/api/client/cluster/clientset/versioned"
+	"open-cluster-management.io/clusteradm/pkg/helpers"
+)
+
+func (o *Options) complete(cmd *cobra.Command, args []string) (err error) {
+	klog.V(1).InfoS("disable options:", "dry-run", o.ClusteradmFlags.DryRun, "names", o.names, "clusters", o.clusters)
+
+	return nil
+}
+
+func (o *Options) validate() (err error) {
+	if o.names == "" {
+		return fmt.Errorf("names is missing")
+	}
+
+	if o.clusters == "" {
+		return fmt.Errorf("clusters is misisng")
+	}
+	return nil
+}
+
+func (o *Options) run() (err error) {
+	addons := sets.NewString()
+	names := strings.Split(o.names, ",")
+	for _, n := range names {
+		if !addons.Has(strings.TrimSpace(n)) {
+			addons.Insert(strings.TrimSpace(n))
+		}
+	}
+
+	clusters := sets.NewString()
+	cs := strings.Split(o.clusters, ",")
+	for _, c := range cs {
```

Comment:
clusters := sets.NewString(cs...)

Reply:
Will this change leads to repeated element, space-prefixed element or space-suffixed element to be inserted into the set if we just use `cs`, which is result of `o.clusters` split by `,`, to initialize the set. 
For example, if the input string is `--clusters cluster1,   cluster2  , cluster1 ,              cluster3`.  will this leads to an  unexpected behavior? 
I also checked the source code of set, https://github.com/kubernetes/apimachinery/blob/v0.22.1/pkg/util/sets/string.go#L30 https://github.com/kubernetes/apimachinery/blob/v0.22.1/pkg/util/sets/string.go#L49 seems that it has no duplicate element check.

Or the cobra framework already did this check for us?

Reply:
sets is implemented by map[key]bool so you will not have duplicated keys

Reply:
> sets is implemented by map[key]bool so you will not have duplicated keys. But you certainly should trim suffix in advance

I just found that if any space exists between names, cobra will throw an error, so we don't need to trim that spaces. 
```
$ clusteradm addon enable --name test211,  test3 ,test1 --cluster cluster1
Deploying test211 add-on to namespaces open-cluster-management-agent-addon of managed cluster: cluster1.
Error: name is required
```